crates/db_views/src/comment_view.rs

```diff
@@ -167,10 +167,9 @@ impl CommentView {
 pub struct CommentQuery<'a> {
   #[builder(!default)]
   pool: &'a DbPool,
-  listing_type: Option<ListingType>,
-  sort: Option<CommentSortType>,
+  listing_type: ListingType,
```

Comment:
Since these aren't optional anymore, you might need to add that `builder !default` line to these two.

edit:

To me it makes sense to leave these optional, and move the `unwrap_or_default` into the query impls.